webapp/src/components/AssetTransactionHistory/AssetTransactionHistory.container.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux'
+
+import { fetchAssetListingHistory } from 'modules/asset//actions'
+import { getAcceptedBidsByAssetFactory } from 'modules/bid/selectors'
+import AssetTransactionHistory from './AssetTransactionHistory'
+
+const mapState = (state, ownProps) => {
+  const getbids = getAcceptedBidsByAssetFactory(ownProps.asset)
```

Comment:
Why the lowercase here?

_I left this comment on modules/bid/selectors but careful this is not working as a memoized selector_